src/lmn-gtm-analytics.js

```diff
@@ -76,7 +76,10 @@ const lmnAnalytics = {
   },
   page: function (category, name, properties, options, callback) {
     ensureSetup();
-    analytics.page.apply(this, argumentsWithEventMetaData(arguments, 2));
+
+    // properties can be ether in the name or properties position
```

Comment:
There's a typo in `either`
Also I would suggest renaming propsPos to `propertyPosition` instead, because dumbdumbs like me might find `propsPos` confusing

But otherwise :shipit: